app/_locales/de/messages.json

```diff
@@ -3732,9 +3732,6 @@
   "noAccountsFound": {
     "message": "Keine Konten für die angegebene Suchanfrage gefunden"
   },
-  "noActivity": {
-    "message": "Noch keine Aktivität"
-  },
```

Comment:
Updating locales to remove unused locale

<img width="466" height="246" alt="Screenshot 2025-09-19 at 2 38 08 PM" src="https://github.com/user-attachments/assets/53c30f1f-3e28-497e-a0cd-1aa1c8c5c309" />